sdk/communication/azure-communication-rooms/src/test/java/com/azure/communication/rooms/RoomsAsyncClientTests.java

```diff
@@ -295,7 +291,8 @@ public void createRoomWithBadParticipantMri(HttpClient httpClient) {
             assertThrows(CommunicationErrorResponseException.class, () -> {
                 roomsAsyncClient.createRoomWithResponse(roomOptions, Context.NONE).block();
             });
-        assertEquals(400, exception.getResponse().getStatusCode());
+        assertEquals("BadRequest", exception.getValue().getError().getCode());
+        assertEquals("Invalid format for communication identifier.", exception.getValue().getError().getMessage());
```

Comment:
Can we just check that the message is not empty? If we change our error message strings on Rooms service, it would cause these tests to fail unnecessarily

Reply:
Updated now